main.go

```diff
@@ -47,6 +48,17 @@ func failf(format string, v ...interface{}) {
 	os.Exit(1)
 }
 
+func getBundlerVersion() (gems.Version, error) {
+	lockFileContent, err := fileutil.ReadStringFromFile("Gemfile.lock")
+	if err != nil {
+		log.Warnf("Could not read from Gemfile.lock, error: %s", err)
+		log.Infof("Using unspecified bundler version")
+		return gems.Version{}, nil
+	}
+
+	return gems.ParseBundlerVersion(lockFileContent)
```

Comment:
is bundler version always specified in gemfile.lock?
not sure if there is an error with `ParseBundlerVersion` then we might better to print the same warning as when failed to read the file itself?

Reply:
AFAIR Case of missing version is handled here: https://github.com/bitrise-steplib/steps-danger/pull/8/files#diff-7ddfb3e035b42cd70649cc33393fe32cR54